common.sh

```diff
@@ -456,6 +456,13 @@ if [[ ! -z ${AGENT_E2E_TEST_SCENARIO} ]]; then
           export MASTER_MEMORY=32768
           export NUM_WORKERS=0
           ;;
+      "TWONODE" )
+          export NUM_MASTERS=2
+          export MASTER_VCPU=4
+          export MASTER_DISK=100
+          export MASTER_MEMORY=32768
+          export NUM_WORKERS=0
```

Comment:
Might want to add the values for ARBITER_VCPU, MEMORY, and DISK here instead of defining that in a separate file.
```
export ARBITER_VCPU=2
export ARBITER_MEMORY=16384
export ARBITER_DISK=100
```

Reply:
Good suggestion, added them in there now, thanks!